beacon_node/eth2-libp2p/src/rpc/handler.rs

```diff
@@ -546,7 +529,8 @@ where
             return Ok(Async::Ready(ProtocolsHandlerEvent::Custom(
                 RPCEvent::Error(
                     *stream_id.get_ref(),
-                    RPCError::Custom("Stream timed out".into()),
+                    // TODO: RPCError::StreamTimeout(¿?)
```

Comment:
Each stream we open has a 10 second timeout. If nothing gets sent in that 10 seconds, the stream has timed out. This error is reporting that no message was received within 10 seconds of asking a request